tcp_check/tox.ini

```diff
@@ -10,7 +10,7 @@ envdir =
     py27: {toxworkdir}/py27
     py38: {toxworkdir}/py38
 description =
-    py{27,38}: e2e ready
+    py38: e2e ready
```

Comment:
why this change?

Reply:
Windows Docker Agent doesn't support Python 2